src/site/assets/img/signin/idme-icon-white.svg

```diff
@@ -1 +1 @@
1.07025.028 0 .05547.0044.08347.013.01188.00362.02233.00973.04112.01967l.01276.0063c.00514.0023.01028.00459.01506.00803.01312.00898.02464.02082.0374.03343l.01046.0105c.0039.00383.00797.00765.0117.01204.0085.01032.01559.02236.02268.03401l.00921.0151c.00638.00993.01259.02005.01826.03209.00336.00707.0062.0149.01418.03553.00815.0216.01648.04337.02268.06706l.00567.02751c.00656.02847.01223.0575.01577.08789l.0023.03324zm2.17934 2.4177a.2772.2772 0 00-.10652-.0256c-.15702-.00553-.26406.06095-.3587.22564-.056.09877-.11112.19926-.1657.29975-.2086.38305-.4241.77871-.73726 1.06223-.42534.38496-1.04102.56512-1.56898.46061-.31404-.06209-.53823-.30701-.67115-.5015-.24386-.35726-.36242-.83832-.34328-1.39274.77713-.0705 3.11951-.4438 3.30578-2.35791.03597-.37102-.06203-.70745-.28356-.97263-.2809-.33624-.72928-.52156-1.26238-.52156-1.56224 0-3.1041 1.76147-3.29958 3.76957-.05334.5508.01028 1.05974.18963 1.5129-.13362.13336-.2598.22946-.3851.29327-.1308.06725-.24422.08081-.32734.03973-.10297-.05158-.14462-.1811-.1611-.28045-.0615-.36777.0179-.79037.11183-1.22691.05547-.256.12477-.5229.18591-.75827.18874-.72618.38422-1.47718.319-2.23584-.05936-.69465-.51164-1.12603-1.18031-1.12603-.93753 0-1.5548.7212-1.95462 1.33943-.0085-.4608-.11041-.79533-.30997-1.01885-.1898-.21283-.46876-.32058-.82888-.32058-.91962 0-1.52928.69274-1.92803 1.29912.00531-.05578.01063-.11271.0163-.16946.02517-.26001.03651-.63256-.15578-.86239-.11466-.13717-.28693-.20652-.51218-.20652-.19885 0-.24847.00115-.46203.03917-.00212.00019-.20877.04394-.28728.1217-.1377.13621-.09393.32401-.06876.43138.00319.01356.00602.02598.00762.03649.01506.10374.01967.22142.01435.35917-.0287.72942-.15436 1.45827-.27363 2.07516-.06433.33243-.137.67058-.2086 1.00434-.15914.7405-.3236 1.50603-.40797 2.27595-.00904.08158.01259.15915.06079.21818.04838.05923.11661.0919.19211.09209l.04963.00038c.50526.00726 1.00185-.01529 1.11882-.32574.1292-.34274.20062-.7596.26389-1.12833l.02782-.16334c.14993-.84787.2809-1.4554.5448-2.23641.13592-.40254.3844-.76515.58537-1.03643.24333-.32822.50562-.64957.80176-.6576.12264-.00687.2024.03249.26265.11865.28764.41362-.07054 1.74866-.2233 2.31875-.03314.12322-.06115.22734-.07887.30376l-.10527.44037c-.15507.64211-.31528 1.306-.39752 1.97754a7.70047 7.70047 0 00-.0225.20747l-.00692.09362.0647.05368c.13043.10909 1.06192-.04356 1.06955-.04604.34346-.12514.41825-.4736.44288-.58805.06398-.29631.12051-.59893.17492-.89123l.0039-.02178c.10261-.5508.2086-1.1207.36739-1.67053.30961-1.06987.72255-1.79986 1.22728-2.16973.21586-.15857.4374-.17614.53806-.04222.17439.23117.0638.85704.01648 1.12432-.06522.37063-.1487.74814-.2295 1.11285l-.00408.01872c-.05175.2325-.10315.46482-.151.6979-.14763.72235-.26885 1.6178.10846 2.13458.19194.26345.48897.3968.88294.3968.41754 0 .81045-.15016 1.23632-.47265.11733-.0896.23571-.19372.3743-.3179.4794.58747 1.02401.83908 1.80965.83908 1.77721 0 2.61265-1.27295 3.08655-2.23048.05973-.12132.12654-.26136.16642-.38019.0592-.17652-.01134-.3691-.16021-.43845z" fill="#FFF"/></g></svg>
```

Comment:
Bad github diff :( I just moved `width="30px" height="12px"` from the `<g>` element to the opening `<svg>` element.

Reply:
VSCode does a _slightly_ better job showing the diff:

![image](https://user-images.githubusercontent.com/20728956/113926962-b4931900-97a1-11eb-954d-ff890ec9d722.png)

Reply:
Yeah it must have been an oversight on my part.